akka-persistence-testkit/src/main/scala/akka/persistence/testkit/internal/Utils.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.testkit.internal
+
+import java.util.{ List => JList }
+
+import akka.annotation.InternalApi
+
+import scala.collection.immutable
+import akka.util.ccompat.JavaConverters._
+import scala.language.implicitConversions
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[testkit] object Utils {
+
+  private[testkit] object JavaCollectionConversions {
+
+    implicit def listConversion[A](jlist: JList[A]): immutable.Seq[A] =
```

Comment:
I don't think we want the magical conversions between java and scala collections. Prefer the more explicit `.asScala` style as in `JavaConverters`.

Is this needed at all, or just use `JavaConverters`?

Reply:
Wow. That's really good point. I think I used conversions, to avoid boilerplate of adding `.asJava`/`.asScala` in all the places.